README

```diff
@@ -12,4 +12,17 @@ autoreconf -vfi
 
 ./configure && make && make install
 
+Building enet -use vcpkg
```

Comment:
This section should be clearly marked as alternative option. It currently feels like this is a mandatory step after (or part of) compiling and installing the library.